src/features/myBeans/BeanListItem.js

```diff
@@ -9,6 +9,7 @@ import CardContent from "@material-ui/core/CardContent";
 import { CardMedia } from "@material-ui/core";
 import Chip from "../common/Chip";
 import Grid from "@material-ui/core/Grid";
+import Image from "../../data/default_beanpic.png";
```

Comment:
```suggestion
import defaultImage from "../../data/default_beanpic.png";
```

コンポーネント以外はCapitalにしないようにしましょう